libs/RobotKit/include/StateMachine.h

```diff
@@ -43,6 +46,20 @@ namespace sm::guard {
 
 }	// namespace sm::guard
 
+namespace sm::action {
+
+	using irc = interface::RobotController;
+
+	struct start_timeout {
+		auto operator()(irc &rc) const { rc.startTimeout(); }
+	};
+
+	struct stop_timeout {
+		auto operator()(irc &rc) const { rc.stopTimeout(); }
+	};
```

Comment:
ici aussi, il faut que les timeout soient à quelque chose de spécifique; j'ai volontairement écrit `goToSleep` pour que ce soit facile à chercher avec cmd+F

```suggestion
	struct start_goToSleep_timeout {
		auto operator()(irc &rc) const { rc.startTimeout(); }
	};

	struct stop_goToSleep_timeout {
		auto operator()(irc &rc) const { rc.stopTimeout(); }
	};
```

Reply:
En terme de style `snake_case` ça ne colle pas trop, c'est pas trop grave?